src/glabs-inline/web/index.scss

```diff
@@ -0,0 +1,81 @@
+@import '_core';
+
+a {
+  text-decoration: none;
+}
+
+h1, p {
+  margin: 0;
+}
+
+.container {
+  font-family: "Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;
+  background-color: $neutral-8;
+  border-top: 2px solid $paid-article-brand;
+
+  display: flex;
+  flex-direction: row;
+  max-width: gs-span(8);
+  height: 110px
+}
+
+.margin {
+  margin: 5px;
+}
+
+.body {
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-basis: 0;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.thumbnail {
+  max-width: 35%;
+  max-height: 110px;
+  overflow:hidden;
+}
+
+.thumbnail__img {
+  height: 100%;
+}
+
+.body__title {
+  font-size: 10px;
+  font-weight: bold;
+  color: $neutral-1;
+
+  margin-bottom: auto;
+
+  @include mq(tablet) {
+    font-size: 12px;
+  }
+}
+
+.body__standfirst {
+  display: none;
+
+  font-size: 10px;
+  color: #333;
+  margin-top: 2px;
+
+  @include mq(tablet) {
+    display: block;
+    font-size: 12px;
+  }
+}
+
+.body__tagline {
+  font-size: 10px;
+  color: $neutral-2;
+  margin-top: 4px;
+  margin-left: auto;
+}
+
+.logo__img {
```

Comment:
Maybe add a small left padding/margin to the img to give it a bit of space from `paid for by`?